roles/filebeat-agent/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+# Installing Filebeat
+
+# Add Elasticsearch apt key
+
+- name: Add Elasticsearch apt key
+  apt_key:
+   url: "{{elasticsearch_repo_gpg_key_url}}"
+   state: present
+
+# Add the Elasticsearch apt repo. For versions 7 of the stack - use '7.x-prerelease':
+
+- name: Adding Elasticsearch repo
+  apt_repository:
+   repo: "deb {{elasticsearch_repo_base_url}}"
+   state: present
+
+# Updateing Repository
+
+- name: "Update cache & Full system update"
+  apt:
+   update_cache: true
```

Comment:
update don't need separate task, use update_cache in package installation

Reply:
done